channels.yaml

```diff
@@ -256,6 +263,19 @@ releases:
     serverArgs: *serverArgs-v3
     agentArgs: *agentArgs-v2
     featureVersions: *featureVersions-v1
+  #  v1.23.11+k3s1 was never released
+  - version: v1.23.12+k3s1
```

Comment:
putting in a PR to remove k3s 1.23.12 and 1.24.6 because we're already releasing 1.23.13 and 1.24.7 as latest versions